.github/workflows/plugins_react_tests.yml

```
@@ -44,17 +61,17 @@ jobs:
       - name: Install Foreman npm dependencies
         run: npm ci --no-audit
         working-directory: ${{ github.workspace }}/projects/foreman
-      - name: Checkout ${{ matrix.repo }}
+      - name: Checkout ${{ matrix.plugin }}
         uses: actions/checkout@v4
         with:
-          repository: ${{ matrix.org }}/${{ matrix.repo }}
-          path: ./projects/${{ matrix.repo }}
-      - name: Generate ${{ matrix.repo }} npm dependencies package-lock
+          repository: ${{ matrix.plugin }}
+          path: ./projects/plugin
```

To be consistent, shouldn't this be:
```suggestion
          path: ${{ github.workspace }}/projects/plugin
```